app/Models/Bot.php

```diff
@@ -21,4 +21,9 @@ public function user()
     {
         return $this->belongsTo(User::class);
     }
+
+    public function scopeByUser($query, $user_id)
+    {
+        return $query->where('user_id', $user_id)->get();
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Models/Bot.php</br>**Line**: 27</br>Variable "user_id" is not in valid camel caps format